package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "scholarxbackend",
```

Shall we change this to scholarx-backend? I think it looks better that way.
```suggestion
  "name": "scholarx-backend",
```

---

Run the npm install command once, so it will update the name on the package-lock.json as well.

---

Done